libcontainer/logs/logs.go

```diff
@@ -0,0 +1,102 @@
+package logs
+
+import (
+	"bufio"
+	"encoding/json"
+	"fmt"
+	"io"
+	"os"
+	"strconv"
+	"sync"
+
+	"github.com/sirupsen/logrus"
+)
+
+var (
+	mutex = &sync.Mutex{}
```

Comment:
This *could* be `var mutex sync.Mutex` (from memory). But this seems okay (though the name could be better).

Reply:
Done